settings/blt.settings.php

```diff
@@ -83,7 +83,7 @@
 /**
  * Acquia Cloud settings.
  */
-if ($is_ah_env && file_exists('/var/www/site-php')) {
+if ($is_ah_env && file_exists('/var/www/site-php' && !$is_acsf)) {
```

Comment:
@grasmash & @lcatlett 
Should this be:
````
if (($is_ah_env && !$is_acsf) && file_exists('/var/www/site-php')) {
````
This line seems like it wouldn't work `file_exists('/var/www/site-php' && !$is_acsf)`?

cc: @rjgwiz